stubs/tree-sitter-languages/METADATA.toml

```
@@ -0,0 +1,5 @@
+version = "1.2.*"
```

```suggestion
version = "1.3.*"
```

It was released two days ago ;)

---

Indeed. That release was because I made a couple pull requests there :)